kitsune/questions/views.py

```diff
@@ -754,7 +708,8 @@ def aaq(request, product_key=None, category_key=None, showform=False,
             url = reverse('questions.details', kwargs={'question_id': question.id})
             return HttpResponseRedirect(url)
 
-        return HttpResponseRedirect(reverse('questions.aaq_confirm'))
+        # What to do here since aaq_confirm is not needed anymore?
+        # return HttpResponseRedirect(reverse('questions.aaq_confirm'))
```

Comment:
How should I handle the following:
```
if request.user.is_active:
    .....
    return HttpResponseRedirect(...)

# What to do here since aaq_confirm is not needed anymore?
return HttpResponseRedirect(reverse('questions.aaq_confirm'))
```
Since a user must be logged in (and active?) to access the AAQ flow, can I just remove the check `request.user.is_active` and always return the `HttpResponseRedirect`?